### PR TITLE
Adding multiweight support to Quantized MobileNetV2 and MobileNetV3

### LINKED ISSUE #4674: Multi-pretrained weights: Add model builders for Quantization
### 🚀 The feature

Update the model builders for all Quantized Classification models:
- [x] GoogleNet
- [x] InceptionV3
- [x] MobileNetV2
- [x] MobileNetV3
- [x] ResNet
- [x] ShuffleNetV2

### PR DESCRIPTION
**Pull request by datumbox**
Fixes #4674

Verified with:
```
python -u train_quantization.py --test-only --weights ImageNet1K_QNNPACK_RefV1 --model mobilenet_v2 --device cpu

python -u train_quantization.py --test-only --weights ImageNet1K_QNNPACK_RefV1 --model mobilenet_v3_large --device cpu
```

cc @bjuncek

### CODE REVIEW

File: torchvision/prototype/models/__init__.py [hunk 1]
```diff
@@ -4,8 +4,7 @@
 from .googlenet import *
 from .inception import *
 from .mnasnet import *
-from .mobilenetv2 import *
-from .mobilenetv3 import *
+from .mobilenet import *
```
Comment (datumbox) [inline torchvision/prototype/models/__init__.py:7]: For BC reasons, we need to maintain the `.mobilenet` space.

File: torchvision/prototype/models/quantization/mobilenetv2.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    if quantize:
+        weights = QuantizedMobileNetV2Weights.verify(weights)
+    else:
+        weights = MobileNetV2Weights.verify(weights)
+
+    if weights is not None:
+        kwargs["num_classes"] = len(weights.meta["categories"])
+        if "backend" in weights.meta:
+            kwargs["backend"] = weights.meta["backend"]
+    backend = kwargs.pop("backend", "qnnpack")
```
Comment (datumbox) [inline torchvision/prototype/models/quantization/mobilenetv2.py:68]: The backend here is `qnnpack`

File: torchvision/prototype/models/quantization/mobilenetv3.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+
+__all__ = [
+    "QuantizableMobileNetV3",
+    "QuantizedMobileNetV3LargeWeights",
+    "mobilenet_v3_large",
+]
+
+
+def _mobilenet_v3_model(
```
Comment (datumbox) [inline torchvision/prototype/models/quantization/mobilenetv3.py:27]: I kept this method instead on dumping everything in the public method because on the future we might want to support also the `_small` version of the model and this method will remain unchanged.

File: torchvision/prototype/models/quantization/mobilenetv3.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        model.fuse_model()
+        model.qconfig = torch.quantization.get_default_qat_qconfig(backend)
+        torch.quantization.prepare_qat(model, inplace=True)
+
+    if weights is not None:
+        model.load_state_dict(weights.state_dict(progress=progress))
+
+    if quantize:
+        torch.quantization.convert(model, inplace=True)
+        model.eval()
```
Comment (datumbox) [inline torchvision/prototype/models/quantization/mobilenetv3.py:54]: Check carefully lines 44-54 comparing to original. This is a simplification of the code.

**Review by datumbox — COMMENTED** [review]
Just a few clarifications to assist review:

**Review by jdsgomes — APPROVED** [review]
Looks good to me!
